tests/integration/helpers.bash

```diff
@@ -419,6 +422,22 @@ function teardown_recvtty() {
 	rm -f "$CONSOLE_SOCKET"
 }
 
+function setup_seccomphook() {
+	cp "$SECCOMPHOOKSOURCE" "$SECCOMPHOOKDESTINATION"
+}
+
+function teardown_seccomphook() {
+	rm -f "$SECCOMPHOOKDESTINATION"
+}
+
+function setup_seccompagent() {
+	"$SECCOMPAGENT" &2> /dev/null
```

Comment:
Could you write a pid file in a temporary directory and then use it instead of pkill below?

Reply:
Done!